backend/Origam.DA.Service/ReferenceIndexManager.cs

```diff
@@ -78,16 +79,33 @@ internal static void UpdateNowOrDeffer(AbstractSchemaItem item)
 
     private static void UpdateNow(AbstractSchemaItem item)
     {
-        referenceDictionary.TryRemove(item.Id, out _);
+        RemoveAllReferencs(item);
         if (!item.IsDeleted)
         {
             Add(item);
         }
     }
 
+    private static void RemoveAllReferencs(AbstractSchemaItem reference)
+    {
+        var referenceInfo = new ReferenceInfo(reference.Id, reference.GetType());
+        var referencelist =  referenceDictionary
```

Comment:
Please fix capitalization of the variable name. Or better yet rename it to referencesToRemove?

Reply:
I ment the variable "referencelist" not the parameter "reference"